eisbuk-admin/package.json

```diff
@@ -35,6 +35,7 @@
     "redux-devtools-extension": "^2.13.8",
     "redux-firestore": "^0.13.0",
     "redux-thunk": "^2.3.0",
+    "ts-jest": "^24.3.0",
```

Comment:
No, it's actually not, some of tests are failing locally for me, due to module name mapper issues. I've solved this by downgrading `ts-jest` to a version compatible with `jest` (v24), as they follow each other on major releases and `CRA` uses `jest@24` under the hood (and since we're using `CRA`, it's a bad idea, and break the functionality if we manually upgrade jest to latest).

Reply:
Right. Got it. Too bad Douglas Crockford didn't include comments in the JSON spec, and now we have to live with a comment free `package.json`.